Logging/Mash.Logging/CentralLogger.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Mash.Logging
+{
+    /// <summary>
+    /// A class that all logging goes through, that hands out logs to each log provider.
+    /// </summary>
+    internal class CentralLogger
```

Comment:
Global instead of central?

Reply:
It isn't global, it is just the one pipe all logging goes through.

At one point I had it named CommonLogger. I thought Central was even better.

I'm open to other suggestions too, though. Luckily, though, this class is not user-facing.